classes/client/s3_client.php

```diff
@@ -335,9 +345,60 @@ public function define_client_section($mform, $config) {
 
         $mform->addElement('select', 's3_region', get_string('settings:aws:region', 'tool_objectfs'), $regionoptions);
         $mform->addHelpButton('s3_region', 'settings:aws:region', 'tool_objectfs');
+
```

Comment:
A large patch has been merged which converts all the settings from formslib to admin settings, this will need to be rebased and this section converted - BUT as I said above, I don't think this patch should add any settings, it should honor the existing moodle proxy settings